crates/fuel-core/src/coins_query.rs

```diff
@@ -709,6 +719,71 @@ mod tests {
             let (owner, asset_ids, base_asset_id, db) = setup_coins_and_messages();
             multiple_assets_helper(owner, &asset_ids, &base_asset_id, db).await;
         }
+
+        mod allow_partial {
+            use crate::{
+                coins_query::tests::{
+                    largest_first::query,
+                    setup_coins,
+                },
+                query::asset_query::AssetSpendTarget,
+            };
+
+            #[tokio::test]
+            async fn largest_first__error_when_not_enough_coins_and_allow_partial_false()
```

Comment:
nit:
```suggestion
            async fn query__error_when_not_enough_coins_and_allow_partial_false()
```
I'm not really sure what the `largest_first` is referring to here`